DEPS

```diff
@@ -6,7 +6,7 @@ vars = {
   'effcee_revision': '2ec8f8738118cc483b67c04a759fee53496c5659',
   'googletest_revision': '3af06fe1664d30f98de1e78c53a7087e842a2547',
   're2_revision': 'ca11026a032ce2a3de4b3c389ee53d2bdc8794d6',
-  'spirv_headers_revision': 'f027d53ded7e230e008d37c8b47ede7cd308e19d',
+  'spirv_headers_revision': '38fe9b08508c876ba830f250a488c51003531e7e',
```

Comment:
NOTE: This will be updated once https://github.com/KhronosGroup/SPIRV-Headers/pull/192 lands.